src/docfx/lib/log/DocfxException.cs

```diff
@@ -22,20 +23,43 @@ public DocfxException(Error error, Exception innerException = null, ErrorLevel?
             OverwriteLevel = overwriteLevel;
         }
 
-        public static bool IsDocfxException(Exception ex, out DocfxException docfxException)
+        public static bool IsDocfxException(Exception ex, out IEnumerable<DocfxException> exceptions)
         {
-            while (ex != null)
+            List<DocfxException> result = null;
```

Comment:
how about initializing with `new List<DocfxException>()` to avoid the null fallback afterwards?